<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mountPartition</class>
 <widget class="QWidget" name="mountPartition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>select mount point path</string>
  </property>
  <widget class="QLabel" name="labelMountPoint">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMount">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Mount</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMountFolder">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Use La&amp;bel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMountReadOnly">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount &amp;Read Only</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShareMountPoint">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Share Mount Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOptions">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pbMountFolder</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBoxMountReadOnly</tabstop>
  <tabstop>checkBoxShareMountPoint</tabstop>
  <tabstop>pbOptions</tabstop>
  <tabstop>pbMount</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
